release-notes/release-notes/5.8-release-notes.md

```diff
@@ -1,39 +1,69 @@
 # 5.8 Release Notes
 
-{% hint style="success" %}
-Cinchy version 5.8 was released on XXXX
-{% endhint %}
+{% hint style="success" %} Cinchy version 5.8 was released on XXXX {% endhint %}
 
 {% hint style="info" %}
 
-For instructions on how to upgrade your platform to the latest version, please review the documentation [here.](../../upgrade-guide/upgrade-guides/)
+For instructions on how to upgrade your platform to the latest version, please
+review the documentation [here.](../../upgrade-guide/upgrade-guides/)
 
 {% endhint %}
+
 ## New Capabilities
 
+### New event-based Source - SOAP API (CDC Triggered)
+
+Cinchy now supports a new Cinchy event-triggered source: SOAP API. This new
+feature initiates a SOAP call based on Change Data Capture (CDC) events
+occurring in Cinchy. The SOAP response then serves as the source for the sync
+and can be mapped to any destination.
+
 ### Connections
 
 ## Enhancements
 
 ### Platform
 
+### Enhanced cache refresh for system entitlements queries
+
+We optimized cache refresh mechanism for system entitlements querying on the
+platform for better performance.
+
+#### Updated POST endpoint for Saved Queries
+
+The POST endpoint for Saved Queries now automatically serializes hierarchical
```

Comment:
This is a breaking change right @gmanreviews? Wouldn't that necessitate more attention on that fact?